transit/models/base.py

```diff
@@ -9,6 +10,7 @@ class BaseModel(models.Model):
     last_modified_by = models.CharField(
         db_column='LastModifiedBy', max_length=255
     )
+    history = HistoricalRecords(inherit=True)
```

Comment:
I wonder if we need to add this also in 'DeliveryStatus' (this model is inherited directly from 'models.Model'). Same regards 'CustomerWeekDays'. I decided no to inherited those models while working on models because those models don't require 'last_modified_date' and 'last_modified_by' etc (based on legacy code).

Reply:
As for the DeliveryStatus it looks rather like enum than business model, but in my opinion CustomerWeekDays could derive from BaseModel

Reply:
You are right about 'DeliveryStatus' - this could be later Enum. 
But CustomerWeekDays is as you wrote typical business model. I think we can add this history into 'CustomerWeekDays' to track history for this entity.